spinta/cli/get.py

```diff
@@ -0,0 +1,39 @@
+import json
+import logging
+from typing import List
+from typing import Optional
+from typer import echo
+from typer import Argument
+from typer import Context as TyperContext
+
+from spinta import commands
+from spinta.cli.helpers.auth import require_auth
+from spinta.cli.helpers.manifest import convert_str_to_manifest_path
+from spinta.cli.helpers.store import prepare_manifest, attach_keymaps
+from spinta.core.context import configure_context
+
+log = logging.getLogger(__name__)
+
+
+def getall(
+    ctx: TyperContext,
+    manifests: Optional[List[str]] = Argument(None, help=("Manifest files to load")),
```

Comment:
I suggest being consistent and using the built-in typehints:

```suggestion
    manifests: list[str] | None = Argument(None, help=("Manifest files to load")),
```

Don't forget to:

```
from __future__ import annotations
```

^ Since spinta supports older versions, that do not support the built-in typehints.

Feel free to ignore this, but I am just mentioning this here, since Karina informed me that's the good practice in the project.